book/website/reproducible-research/rdm/rdm-sharing.md

```diff
@@ -3,44 +3,31 @@
-Existing policies on data archiving notwithstanding, in today’s practice data are primarily stored in private files, not in secure institutional repositories, and are effectively lost to the public.
-This lack of access to scientific data should be removed for several reasons:
+There are many reasons to share your research data publicly. 
 
 1. To allow the possibility to fully reproduce a scientific study.
 2. To prevent duplicate efforts and speed up scientific progress.
 Large amounts of research funds and careers of researchers can be wasted by only sharing a small part of research in the form of publications.
 3. To facilitate collaboration and increase the impact and quality of scientific research.
 4. To make results of research openly available as a public good, since research is often publicly-funded.
 
-Accordingly, there is an ongoing global data revolution that seeks to advance collaboration and the creation and expansion of effective, efficient research programs.
-Sharing data openly is crucial to meeting these objectives.
-Open data means that the data is freely available on the internet permitting any user to download, copy, analyse, re-process, and re-use it for any other purpose without financial, legal, or technical barriers, other than those inseparable from gaining access to the internet itself.
-This represents a real shift in how research works.
-At the moment, anyone who wishes to use scientific data from a researcher often has to contact that researcher and make a request.
-However, "open by default" presumes a publication for all.
-Researchers need to justify why data is kept closed.
-Such justification may be security or data protection reasons, for example.
-Free access to, and subsequent use of, data is of significant value to society and the economy, and that data should, therefore, be open by default.
-Research is often publicly-funded, so the results of this research should be openly available as a public good.
+You can read more about why data should be available, and why some data should remain closed, in the {ref}`Open Data section <rr-open-data>`
 
 (rr-rdm-sharing-steps)=
 ## Steps To Share Your Data
 
 ### Step 1: Select what data you want to share
 
-Not all data can be made openly available, due to ethical and commercial concerns, and you may decide that some of your intermediate data is too large to share.
+Not all data can be made openly available, due to ethical and commercial concerns (see the {ref}`Open Data section <rr-open-data>), and you may decide that some of your intermediate data is too large to share.
 As such, you first need to decide which data you need to share for others to be able to reproduce your research.
 
 ### Step 2: Choose a data repository or other sharing platform
 
 Data should be shared in a formal, open, and indexed data repository where possible so that it will be accessible in the long-run.
```

Comment:
```suggestion
Data should be shared in a formal, open, and indexed data repository where possible so that it will be accessible in the long run.
```